packages/frontend/src/components/info-popover/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { Popover } from "@carrot-kpi/ui";
+import React, { ReactNode, useCallback, useRef, useState } from "react";
+import { ReactComponent as Info } from "../../assets/info.svg";
+
+interface InfoPopoverProps {
+    children?: ReactNode;
+}
+
+export const InfoPopover = ({ children }: InfoPopoverProps) => {
```

Comment:
Maybe we can move this component to the `UI` lib, I think it can be useful if we need to add hints to the inputs in the creation form for example, wdyt?

Reply:
I thought about that but I'd like to add some functionality before sending it to the UI lib. Namely, I'd like to turn this to a generalized component with `info`, `warning` and `error` variants with dedicated icons/colors and such. I'll add a `TODO` though.